### PR TITLE
Enable the whole logcat log

### PR DESCRIPTION
**Pull request by keyonghan**
Help with https://github.com/flutter/flutter/issues/142572.

### CODE REVIEW

**Review by zanderso — APPROVED** [review]

File: dev/devicelab/lib/framework/devices.dart [hunk 1]
```diff
@@ -748,7 +748,7 @@ class AndroidDevice extends Device {
       // For local testing, this can just be:
       //   <String>['-s', deviceId, 'logcat']
       // to view the whole log, or just run logcat alongside this.
-      <String>['-s', deviceId, 'logcat', 'ActivityManager:I', 'flutter:V', '*:F'],
+      <String>['-s', deviceId, 'logcat'],
```
Comment (matanlurey) [inline dev/devicelab/lib/framework/devices.dart:751]: Is the entire comment stream above stale now?

**Review by matanlurey — CHANGES_REQUESTED** [review]
There is a `test` folder that includes commands that run against a fake ADB, is there a reason this can't be tested?

**Comment by keyonghan** [comment]
> There is a `test` folder that includes commands that run against a fake ADB, is there a reason this can't be tested?

This is simply an `adb` command change (less args). Not changing any logics but include more logs in the existing log file.
I can try adding a mock process manager and a mock process, mocking that the process does get executed. But I am not sure if we are really testing any logic. Please advise if you think we need to test from a different perspective.

**Comment by matanlurey** [comment]
Ok, I'm convinced. LGTM and test-exempt.

**Review by matanlurey — APPROVED** [review]

**Pull request merged**